docker/release/dockerfiles/opensearch.al2.dockerfile

```diff
@@ -57,7 +57,7 @@ ARG OPENSEARCH_HOME=/usr/share/opensearch
 # Update packages
 # Install the tools we need: tar and gzip to unpack the OpenSearch tarball, and shadow-utils to give us `groupadd` and `useradd`.
 # Install which to allow running of securityadmin.sh
-RUN yum update -y && yum install -y tar gzip shadow-utils which && yum clean all
+RUN yum update -y && yum clean all && yum install -y tar gzip shadow-utils which openssl-devel openssl && yum clean all
```

Comment:
Why do you need to clean all then install again?

Reply:
Actually it doesn't clean and install all again, but just clears caches and metadata to keep the docker image size lean. Nvm, I have removed this now since i did not find any reduction in image size.